$NetBSD: patch-ca,v 1.1.6.1 2008/09/23 11:15:53 tron Exp $

Before calling sigsend(), make sure the function actuall exists. Fixes
the build on NetBSD >= 4.99.49, in which the macro P_MYID is defined
somewhere unrelated to signals.

--- configure.in.orig	2008-08-04 15:19:07.000000000 +0300
+++ configure.in	2008-09-16 13:52:59.000000000 +0300
@@ -2001,7 +2001,7 @@
 paths.h semaphore.h)
 
 # Already-done: strcasecmp
-AC_CHECK_FUNCS(lstat putenv select setenv setlocale strcoll tcgetattr)
+AC_CHECK_FUNCS(lstat putenv select setenv setlocale sigsend strcoll tcgetattr)
 
 AC_STAT_MACROS_BROKEN
 MYSQL_SIGNAL_CHECK
